pkg/virtctl/imageupload/imageupload.go

```diff
@@ -313,25 +307,25 @@ func (c *command) run(args []string) error {
 		return fmt.Errorf("cannot obtain KubeVirt client: %v", err)
 	}
 
-	pvc, err := getAndValidateUploadPVC(virtClient, namespace, name, noCreate, archiveUpload)
+	pvc, err := getAndValidateUploadPVC(virtClient, namespace, c)
```

Comment:
Make `getAndValidateUploadPVC` a method of `command` instead of passing `c` as argument.

Reply:
I thought about it doing it. @0xFelix Do you think we should consider these functions to be part of command? It's not validate arguments of the command or something like this.

Reply:
You need the context in these functions IOW you want to access data of the struct. That's valid to me.